src/__tests__/resolve-styles-test.js

```diff
@@ -621,6 +621,59 @@ describe('resolveStyles', function() {
       });
     });
   });
+  
```

Comment:
You've got a few eslint warnings in this file. I think we might make those errors at some point, but it would be nice to go ahead and clean then up here.

```bash
/home/travis/build/FormidableLabs/radium/src/__tests__/resolve-styles-test.js
  624:1   warning  Trailing spaces not allowed            no-trailing-spaces
  634:26  warning  Props should be sorted alphabetically  react/jsx-sort-props
  650:1   warning  Trailing spaces not allowed            no-trailing-spaces
  660:26  warning  Props should be sorted alphabetically  react/jsx-sort-props
```